MailManager/src/main/java/inc/softserve/health/MailManagerHealthCheck.java

```diff
@@ -15,30 +17,46 @@
     private String espConnectorAdminUrl;
     private String espConnectorHealthPath;
     private EnvelopeTools envelopeTools;
+    private MongoManaged mongo;
     private ObjectMapper objectMapper = new ObjectMapper();
 
     @Inject
     public MailManagerHealthCheck(@ESPConnectorAdmin String ESPConnectorAdminUrl,
                                   @ESPConnectorHealth String ESPConnectorHealthPath,
-                                  EnvelopeTools envelopeTools){
+                                  EnvelopeTools envelopeTools, MongoManaged mongo){
         this.espConnectorAdminUrl = ESPConnectorAdminUrl;
         this.espConnectorHealthPath = ESPConnectorHealthPath;
         this.envelopeTools = envelopeTools;
+        this.mongo = mongo;
     }
 
     protected Result check() throws Exception {
-        String responseAsString = envelopeTools.sendGetRequest(espConnectorAdminUrl, espConnectorHealthPath);
         boolean espConnectorHealthy = false;
+        StringBuilder unhealthyMessage = new StringBuilder("");
+
         try {
+            String responseAsString = envelopeTools.sendGetRequest(espConnectorAdminUrl, espConnectorHealthPath);
             JsonNode responseAsNode = objectMapper.readValue(responseAsString, JsonNode.class);
             espConnectorHealthy = responseAsNode.get("health").get("healthy").asBoolean();
-        } catch (IOException e) {
+        } catch (Exception e) {
             e.printStackTrace();
         }
-        if (espConnectorHealthy) {
-            return Result.healthy();
+
+        if (!espConnectorHealthy) {
+            unhealthyMessage = unhealthyMessage.append("ESPConnector is unavailable!");
+        }
+
+        try{
+            DB db = mongo.getDb();
+            db.getStats();
+        }catch(Exception e){
+                  unhealthyMessage = unhealthyMessage.append(" Mongo DB is unavailable!");
+        }
+
+        if(unhealthyMessage.length() != 0){
+            return Result.unhealthy(unhealthyMessage.toString());
         }
```

Comment:
There are minor spacing issues withing those 10 lines of code. Could you please fix it before merging?